### PR TITLE
Roe 99 add page data into session when posting

### PR DESCRIPTION
**Pull request by mwestacott**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-99
https://companieshouse.atlassian.net/browse/ROE-100

### Change description

* Store data in session when page is submitted
* Retrieve and display data from session when page is displayed
* Also fix minor issue on Beneficial Owner Individual screen, where option button choices weren't being set

### Work checklist

- [X] Tests added where applicable
- [X] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/managing.officer.controller.ts
Comment (markpit) [inline src/controllers/managing.officer.controller.ts:8]: should have a try -catch around the get function (like post)

Comment (markpit) [reply src/controllers/managing.officer.controller.ts:8]: and a test that it handles teh error (see other controller tests)

File: test/controllers/managing.officer.controller.spec.ts
Comment (mattch1) [inline test/controllers/managing.officer.controller.spec.ts:30]: We can test the data values here - see the tests for corporate managing officers.

File: src/model/managing.officer.model.ts
Comment (markpit) [inline src/model/managing.officer.model.ts:12]: remove reference to Entity in comment

File: src/controllers/managing.officer.controller.ts
Comment (mwestacott) [reply src/controllers/managing.officer.controller.ts:8]: Agree. Weirdly, test is there and passing so will have a look.

File: test/controllers/managing.officer.controller.spec.ts
Comment (markpit) [inline test/controllers/managing.officer.controller.spec.ts:32]: I'd add a couple of checks to see if session data appeared on the page

**Review by mattch1 — APPROVED** [review]
I will approve but I will advise waiting for Mark to approve as well before merging

**Review by markpit — APPROVED** [review]

**Pull request merged**
